synapseclient/models/folder.py

```diff
@@ -266,6 +312,78 @@ async def delete(self, synapse_client: Optional[Synapse] = None) -> None:
                 lambda: Synapse.get_client(synapse_client=synapse_client).delete(
                     obj=self.id,
                 ),
+                current_context=current_context,
+            ),
+        )
+
+    @otel_trace_method(
+        method_to_trace_name=lambda self, **kwargs: f"Folder_Copy: {self.id}"
+    )
+    async def copy(
+        self,
+        destination_id: str,
```

Comment:
Same note above about wanting to be consistent in the future and not have destination id

Reply:
Updated in latest commit: https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1061/commits/33a1f4721bd7b6c4168d107cfcc0fb257304520b